.github/ISSUE_TEMPLATE/question.yml

```
@@ -0,0 +1,17 @@
+name: ❓ Question
```

I would like to have this more generic,we have now bug and improvements and that is for everything else, so something like:
```suggestion
name: ❓ Question or general issue
```
maybe?

---

If we vote for "DIscussions" then we can remove this at all.

---

hmm not really, I could imagine someone having an issue but it is not a bug or a feature request